blueprints/smart/files/__root__/__routepath__/__route__/__dumb__/__name__/__name__.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+type Props = {
```

Comment:
What is the purpose of the `type` keyword here? Could not find anything on Google about it.